### PR TITLE
Input

### PR DESCRIPTION
**Pull request by Leo6016**
added input function and a test function for input.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,3 @@
 [Leo Xu](https://github.com/Leo6016) \
 [Otis Lu](https://github.com/OtisL99) \
 [Danilo Montes](https://github.com/danilo-montes)
-
-## PyPI Page
-[Package on PyPI]()
```
Comment (danilo-montes) [inline README.md:16]: was this meant to be deleted? it's a boilerplate for when we publish later in the project

Comment (Leo6016) [reply README.md:16]: No I may have merged the wrong version, please don't delete

File: src/thisday/thisday.py [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+def get_events(soup):
+    my_data = []
+    events= soup.select('li.event')
+
+    for event in events:
+        my_data.append(event.get_text())
+
+    return my_data
+    
+def ProcessInput(inputString):
```
Comment (danilo-montes) [inline src/thisday/thisday.py:21]: i think convention for function names is lowercase with underscores, like `process_input`

File: tests/test_thisday.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import pytest
-from thisday import thisday
+from thisday import thisday,ProcessInput
```
Comment (danilo-montes) [inline tests/test_thisday.py:2]: this import should just be the package, and then when functions are tested, you use `thisday.<function>`

File: src/thisday/thisday.py (hunk 2 above)
Comment (Leo6016) [reply src/thisday/thisday.py:21]: Ok I'll change that and create another request

File: tests/test_thisday.py (hunk 3 above)
Comment (Leo6016) [reply tests/test_thisday.py:2]: Ok

File: tests/test_thisday.py [hunk 4]
```diff
@@ -19,4 +19,11 @@ def test_sanity_check(self):
         expected = True
         actual = True
         assert actual == expected, 'Expected True to equal True'
+
+    def test_input():
```
Comment (danilo-montes) [inline tests/test_thisday.py:23]: the function name must follow pytest convention in order to work, so it should be `test_process_input`

**Comment by Leo6016** [comment]
Will change some stuff and pull again

**Pull request closed without merging**